### DISCUSSION TITLE
[express] Locals type error for @types/express in 4.17.16

### DISCUSSION THREAD

**Discussion by rrnara** [DefinitelyTyped/DefinitelyTyped#64085] [Issues with a @types package]
With the latest release I see the following error:
```
node_modules/@types/express/index.d.ts:112:35 - error TS2694: Namespace '"/Users/rrnara/Work/test/node_modules/@types/express-serve-static-core/index"' has no exported member 'Locals'.

112     interface Locals extends core.Locals {}
                                      ~~~~~~
```

[a comment by an automated account is not shown]

**Reply by sudopriestmx** [reply]
To anyone currently affected by this bug, until it's fixed, a solution is to install the ' "@types/express-serve-static-core"' dependency separately on your devDependencies using version "^4.17.33", which fixes this problem, until the peer dependency is updated on the base '@types/express' package
